src/jemalloc.c

```diff
@@ -868,6 +868,14 @@ malloc_slow_flag_init(void) {
 	malloc_slow = (malloc_slow_flags != 0);
 }
 
+/*
+ * One of the CONF_MIN macros below expands, in one of the use points,
+ * to "usigned integer < 0", which is always false, triggering the
```

Comment:
I couldn't reproduce this locally but it happened on CI so it was hard for me to nail down exactly which call to `CONF_MIN` was causing this. I will try to move this to the single call where this happens.

Reply:
Due to the way the code below is structured, its really hard to tell where the actual `CONF_#..._#...` macro that expands to the code with the issue even is, and in which instance of the macro invokation it happens :/ 

I've limited it a bit more, and will send a couple of commits limiting it a bit further, but since I cannot reproduce this locally its going to take a while. Expect the PR to be in a semi-broken state until then.